IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormCommitTests.cs

```diff
@@ -207,6 +208,91 @@ public void Should_handle_well_commit_message_in_commit_message_menu()
             });
         }
 
+        [Test]
+        public void Should_stage_only_filtered_on_StageAll()
+        {
+            _referenceRepository.Reset();
+            _referenceRepository.CreateRepoFile("file1A.txt", "Test");
+            _referenceRepository.CreateRepoFile("file1B.txt", "Test");
+            _referenceRepository.CreateRepoFile("file2.txt", "Test");
+
+            RunFormTest(async form =>
+            {
+                using (var cts = new CancellationTokenSource(AsyncTestHelper.UnexpectedTimeout))
+                {
+                    await ThreadHelper.JoinPendingOperationsAsync(cts.Token);
+                }
+
+                Assert.AreEqual("Stage All", form.GetTestAccessor().StageAllToolItem.ToolTipText);
```

Comment:
Asserting against strings isn't a good idea, a lot of our contributors run on non-English environments, and may have different locale and localisation settings.
If you need to assert against default strings, please make sure to run tests in en-US culture:
```cs
    [SetCulture("en-US")]
    [SetUICulture("en-US")]
```